src/infrastructure/storage/schema.js

```diff
@@ -112,11 +112,12 @@ const tasksRelations = relations(tasks, ({ one, many }) => ({
   }),
   user: one(users, {
     fields: [tasks.id_user],
-    refereces: [users.id_user]
+    references: [users.id_user]
   }),
   parentTask: one(tasks, {
     fields: [tasks.id_parent_task],
     references: [tasks.id_task],
+    nullable: true,
```

Comment:
ako si to ja predstavujem tak ked bude ParentTask null tak to znamena ze to je normalny task, ked bude parentTask 1 tak to je subtask tasku 1 napr.

Reply:
presne tak, ja myslim ze to je nullable i bez teto zmeny ne? ale klidne to tam nechme takto explicitne :)

Reply:
idk chcel som fixnut ten schema error ale nepomohlo :D